toml-rb.gemspec

```diff
@@ -2,22 +2,22 @@ Gem::Specification.new do |s|
   s.name        = 'toml-rb'
   s.version     = '0.1.6'
   s.date        = Time.now.strftime('%Y-%m-%d')
-  s.summary     = "TOML parser in ruby, for ruby."
-  s.description = "A TOML parser using Citrus parsing library. "
-  s.authors     = ["Emiliano Mancuso", "Lucas Tolchinsky"]
+  s.summary     = 'TOML parser in ruby, for ruby.'
+  s.description = 'A TOML parser using Citrus parsing library. '
+  s.authors     = ['Emiliano Mancuso', 'Lucas Tolchinsky']
   s.email       = ['emiliano.mancuso@gmail.com', 'lucas.tolchinsky@gmail.com']
   s.homepage    = 'http://github.com/eMancu/toml-rb'
-  s.license     = "MIT"
+  s.license     = 'MIT'
 
   s.files = Dir[
-    "README.md",
-    "Rakefile",
-    "lib/**/*.rb",
-    "lib/**/*.citrus",
-    "*.gemspec",
-    "test/*.*",
-    "init.rb"
+    'README.md',
+    'Rakefile',
+    'lib/**/*.rb',
+    'lib/**/*.citrus',
+    '*.gemspec',
+    'test/*.*',
+    'init.rb'
```

Comment:
I don't think it's a big deal, but I started using double quotes everywhere to avoid having to think about it. Also, this happened: http://viget.com/extend/just-use-double-quoted-ruby-strings